src/PIL/Image.py

```diff
@@ -1049,7 +1049,7 @@ def convert_transparency(m, v):
                 new_im.info['transparency'] = trns
         return new_im
 
-    def quantize(self, colors=256, method=None, kmeans=0, palette=None):
+    def quantize(self, colors=256, method=None, dither=1, kmeans=0, palette=None):
```

Comment:
Please add the new parameter to the end (and in the docstring).

Some people may already be calling it with positional rather than named arguments, and we don't want to break the API for them.